<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="configPushButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开设置</string>
    </property>
   </widget>
   <widget class="QWidget" name="configWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QTabWidget" name="tabWidget_config">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>981</width>
       <height>351</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>常用设置</string>
      </attribute>
      <widget class="QLabel" name="label_studentNumber">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>学生数量</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_studentNumber">
       <property name="geometry">
        <rect>
         <x>870</x>
         <y>20</y>
         <width>42</width>
         <height>41</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_saveConfigs">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>400</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始？？</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>981</width>
      <height>441</height>
     </rect>
    </property>
    <widget class="QListWidget" name="listWidget_studentNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>211</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>configWidget</zorder>
   <zorder>configPushButton</zorder>
   <zorder>startPushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
